### ISSUE TITLE
move MovementDescriber to scener-phet

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#133]
I think we should do this now rather than later. I'd be curious to see if @mbarlow12 could utilize this in his magnet describer code in FL.

Once it is over in scenery-phet, we should factor out strings into that string js file.

**Comment by zepumph** [comment]
I moved the describer out to scenery-phet in the above commits. @jessegreenberg please see if you can add it BASE. Not a high priority, but it would be good to be consistent. 
@mbarlow12 perhaps you can think of a way that FL could use this. Maybe you could wire it up with the bounds2 and locationProperty and it could monitor whether the magnet is "atEdge" such that you could add that text to the PDOM. 

Not sure if these warrant new issues, I'll leave it up to you.

**Comment by zepumph** [comment]
In https://github.com/phetsims/scenery-phet/commit/3d56cdef44f5358a8d749088a353e6d4df1645d3 I did quite a bit more work on these describers. Perhaps a reviewish before integrating it would be good. I don't want to break anyone's sim.

**Comment by jessegreenberg** [comment]
I had forgotten that the way direction is described in BASE is that the alert changes depending on whether the balloon is grabbed or moving independently. While grabbed, the alert is just "Left." But while independent, the alert might be "Yellow Balloon, now left". Or it might contain other information like "Yellow Balloon moves extremely slowly left." @zepumph would it be OK if MovementDescriber had functions to return the direction string without announcing them so the client could insert into their own string patterns and alert as necessary?

Or we could have a DirectionDescriber that handles generation of just these direction strings, MovementDescriber could use this to generate the strings and send them to utteranceQueue. Cases like Friction and Faradays Law (that don't have independent movement) could MovementDescriber directly this while BASE could just use DirectionDescriber.

**Comment by zepumph** [comment]
>Or we could have a DirectionDescriber that handles generation of just these direction strings, MovementDescriber could use this to generate the strings and send them to utteranceQueue.

We would definitely want to rename one of these such that `Describer` was a "thing" (probably that just dealt with text), and we had another suffix for the other thing (likely something that alerts, like `Alerter`).

Unless there are any objections, how about we solidify that differentiation (I'll make an issue https://github.com/phetsims/scenery-phet/issues/417). For now Describers format and formulate text, most often by looking at the model, but can be static as well. Alerters actually call over to `utteranceQueue`.


>Cases like Friction and Faradays Law (that don't have independent movement) could MovementDescriber directly this while BASE could just use DirectionDescriber.

I thought @mbarlow12 mentioned that he wished that there could just be a text-only describer for movement (or maybe it was the border stuff), so likely what you are talking about sounds useful in many cases.

My recommendation is to go crazy! Please keep friction in a functional state, but otherwise organize however you think is best.

Here is a bit more details about one thought.

Current hierarchical structure:

- MovementDescriber (really an alerter)
     - BorderAlertsDescriber (really an alerter)

But perhaps it should be more split up, perhaps this is too verbose, but what if it was more like:
- MovementAlerter
    - MovementDescriber
    - BorderAlerter
         - BorderDescriber

Any you could potentially use any of these that suited your needs. That may be hard to implement from our current implementation, honestly I'm not sure, and would take some investigation to be sure. Would these be useful?

**Comment by jessegreenberg** [comment]
> But perhaps it should be more split up, perhaps this is too verbose, but what if it was more like:

Agreed, that sounds best for similar reasons to this separation.

> My recommendation is to go crazy!

🚀 Thanks!
